conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceHandlerGenerator.java

```diff
@@ -91,6 +91,7 @@
     private static final String RUNTIME_VAR_NAME = "runtime";
     private static final String DESERIALIZER_VAR_NAME = "deserializer";
     private static final String SERIALIZER_VAR_NAME = "serializer";
+    private static final String DEPRECATED_CALLBACK_VAR_NAME = "deprecation";
```

Comment:
deprecation().deprecated

Reply:
actually I renamed every deprecatiated to depreciation. So this would be `deprecation.deprecation`.

Is it bad?

Reply:
I think `deprecation()` gives us the interface which handles functionality related to deprecated things, the `deprecated(args)` method marks an endpoint as being deprecated. That's how I modeled it in my head anyhow.